packages/menus/src/elements/Menu.js

```diff
@@ -18,6 +19,18 @@ import MediaItem from '../views/items/media/MediaItem';
 import NextItem from '../views/items/NextItem';
 import PreviousItem from '../views/items/PreviousItem';
 
+const ScrollableMenuView = styled(MenuView)`
+  /* stylelint-disable */
+  ${({ maxHeight }) =>
+    maxHeight
+      ? `
+    max-height: ${maxHeight};
+    overflow-y: scroll;
```

Comment:
this should probably just be `overflow: auto`

Reply:
Fixed.